### DISCUSSION TITLE
Touchscreen Enable for T460s Big Sur

### DISCUSSION THREAD

**Discussion by askwakhid** [duszmox/Thinkpad-T460s-macOS-OpenCore#5] [Q&A]
Hi,

Do you have suggestions for enable touchscreen in Big Sur ?
I see the touch interface using I2C.

Thank you.

**Comment by mhl221135** [comment]
I tried early but nothing to show you ) touchscreen not detecting even in devices also i2chid kext not helped with it

**Comment by duszmox** [comment]
Sadly, my laptop isn't a touchscreen model, and I don't have access to any other 6th gen touchscreen Thinkpad.  From what I could find online about touchscreen hackintoshes, most of them use VoodooI2c and VoodooI2CHID kexts to achieve the touchscreen, but I couldn't find any sources that described that they managed to get it work on a T series laptop.  But hopefully someday we could enable it via some new development
